INSTALLATION.md

```diff
@@ -1,87 +1,63 @@
 
-* [Introduction](#introduction)
-* [Desktop](#desktop)
-  * [MacOS](#macos)
-  * [Linux](#linux)
-  * [Windows](#windows)
+* [Windows Installation](#windows-installation)
+* [MacOS Installation](#macos-installation)
+* [Debian Linux Installation](#debian-linux-installation)
 
-## Introduction
+> **Note:** The **MassaStation** is currently available for Windows, MacOS, and Debian Linux. Support for other Linux distributions will be added in the future.
 
-From this point you'll start to custom your Massa's journey thanks to Thyra. If you want to learn more about Thyra's objectives and mindset, go here; if you're curious about the technical aspects and want to contribute go [there](./CONTRIBUTING.md).
+> **Troubleshooting:** If you encounter any issues during the installation process, do not hesitate to [open an issue](https://github.com/massalabs/thyra/issues/new) on GitHub.
 
-This document will guide you through the installation process and the first steps to get started with Thyra.
+## Windows Installation
 
+1. Download the latest version of **MassaStation** installer for Windows from the [official website](https://github.com/test/projectX/releases/latest/download/project-installer_Windows_amd64.msi).
+2. Locate the downloaded `.msi` installer file and double-click on it to start the installation process.
+3. Follow the on-screen instructions to proceed with the installation.
+4. Once the installation is complete, you will see a confirmation message. Click "Finish" to exit the installer.
 
-> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
+Congratulations! You have successfully installed the **MassaStation** on your computer. You can start MassaStation by searching for it in the Start menu or by clicking on the icon on your desktop (if you chose to create one during the installation).
 
-## Instructions by Devices
+## MacOS Installation
 
-By following these instructions, you will: 
-* Install Thyra to access and perform actions on the blockchain. 
-* Install Thyra application to interact and use Thyra and its plugins very easily.
+1. Download the latest version of **MassaStation** installer for MacOS depending on your CPU architecture:
 
-### Desktop
+   - For Intel-based Macs (i5, i7, etc.), download the installer for `amd64` architecture from [here](https://github.com/test/projectX/releases/latest/download/project-installer_MacOS_amd64.pkg).
+   - For Apple Silicon-based Macs (M1, M2, etc.), download the installer for `arm64` architecture from [here](https://github.com/test/projectX/releases/latest/download/project-installer_MacOS_arm64.pkg).
 
+2. Locate the downloaded `.pkg` installer file and double-click on it to initiate the installation.
+3. If prompted, enter your administrator password to authorize the installation.
```

Comment:
@thomas-senechal You don't have to enter your admin password, but you have to "open anyway" the installer because it's coming from an unknown developer.
Normally mac users are used to this as it's the same for Brave or Discord for instance. But still let's mentioned it. 
"If prompted, enter your administrator password or go to your security & privacy panel to authorise the installation" 

WDYT?